### ISSUE TITLE
note for TiDB Vector Search

### ISSUE THREAD

**Issue by shawn0915** [pingcap/docs#19373]
File: [/release-7.5/tidb-cloud/vector-search-get-started-using-python.md](https://docs.pingcap.com/tidbcloud/vector-search-get-started-using-python)

> TiDB Vector Search is currently in beta and only available for TiDB Cloud Serverless clusters.

we need to update this content cause tidb 8.4 release?

**Comment by Oreoxmt** [comment]
@shawn0915 Thank you for bringing this up. Let me clarify:

The document at https://docs.pingcap.com/tidbcloud/vector-search-get-started-using-python is specifically for TiDB Cloud users. The "beta" status and "only available for TiDB Cloud Serverless clusters" statement refers to the feature's availability within TiDB Cloud - meaning it's not yet supported on TiDB Cloud Dedicated clusters.

If you're interested in Vector Search documentation for TiDB v8.4, please refer to the TiDB Self-Managed documenattion at https://docs.pingcap.com/tidb/dev/vector-search-get-started-using-python.

**Comment by shawn0915** [comment]
@Oreoxmt thanks for reply.

> it's not yet supported on TiDB Cloud Dedicated clusters.

It is better to clarify the scope or the description like "available for TiDB Self-Managed clusters and [TiDB Cloud Serverless]" in both sides?

- TiDB

https://docs.pingcap.com/tidb/v8.4/vector-search-get-started-using-python

> The vector search feature is only available for TiDB Self-Managed clusters and [TiDB Cloud Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-cloud-serverless) clusters.

- TiDB Cloud

https://docs.pingcap.com/tidbcloud/vector-search-get-started-using-python

> TiDB Vector Search is currently in beta and only available for [TiDB Cloud Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-cloud-serverless) clusters.

BTW,

in tidb docs, "vector search" should be "Vector Search"?

**Comment by Oreoxmt** [comment]
@shawn0915 

- This TiDB document [docs.pingcap.com/tidb/v8.4/vector-search-get-started-using-python](https://docs.pingcap.com/tidb/v8.4/vector-search-get-started-using-python) explicitly mentions "only available for TiDB Self-Managed clusters and TiDB Cloud Serverless clusters" because it includes usage examples for both TiDB Self-Managed and TiDB Cloud Serverless clusters.

> in tidb docs, "vector search" should be "Vector Search"?

- Regarding capitalization:
@qiancai Could you please confirm the preferred terminology? I notice that both "vector search" and "Vector Search" appear in the TiDB Cloud documentation. Should we standardize on one format for consistency?

**Comment by breezewish** [comment]
Maybe we need to change occurance to this everywhere, like:

TiDB Vector Search is currently in beta and is not available for TiDB Cloud Dedicated clusters.

This could lead to less confusions.

**Comment by qiancai** [comment]
> @shawn0915
> 
> * This TiDB document [docs.pingcap.com/tidb/v8.4/vector-search-get-started-using-python](https://docs.pingcap.com/tidb/v8.4/vector-search-get-started-using-python) explicitly mentions "only available for TiDB Self-Managed clusters and TiDB Cloud Serverless clusters" because it includes usage examples for both TiDB Self-Managed and TiDB Cloud Serverless clusters.
> 
> > in tidb docs, "vector search" should be "Vector Search"?
> 
> * Regarding capitalization:
>   @qiancai Could you please confirm the preferred terminology? I notice that both "vector search" and "Vector Search" appear in the TiDB Cloud documentation. Should we standardize on one format for consistency?

Yes, I've created a PR https://github.com/pingcap/docs/pull/19385 to standardize the capitalization of “vector search” and “Vector Search” across the TiDB Cloud documentation to improve consistency and readability. The changes follow these rules:

	•	"TiDB Vector Search": Used when referring to TiDB’s specific product feature.
	•	"vector search": Used when referring to the concept of vector search in a general or technical context.

**Comment by breezewish** [comment]
I guess a recent misunderstanding caused by current Serverless doc is here: https://github.com/pingcap/autoflow/issues/399
